plugins/hls-fourmolu-plugin/test/testdata/Fourmolu.formatted.hs

```diff
@@ -10,7 +10,7 @@ foo 3 = 2
 foo x = x
 bar :: String -> IO String
 bar s = do
-    x <- return "hello"
-    return "asdf"
+  x <- return "hello"
+  return "asdf"
```

Comment:
This is a bit odd. I don't think Fourmolu has started defaulting to two-space indentation...

Reply:
Pretty sure this is my mistake, I have a local `fourmolu.yaml` where I default to two-space indentation. The tests are probably picking this up.

Reply:
This confused me a great deal, which is why I initially updated the tests, realised my mistake after a CI run, and then apparently restored the wrong golden files.

Reply:
Okay, as long as CI was correctly failing then that's all good.

Reply:
CI caught it :)

Reply:
However, the tests should be isolated enough that my environment shouldn't bleed into it. Presumably, fourmolu tests should be migrated to the new testing infrastructure where everything is in a temp directory.